src/networking.c

```diff
@@ -319,6 +319,44 @@ int prepareClientToWrite(client *c) {
     return C_OK;
 }
 
+/* Returns everything in the client reply linked list in a SDS format. */
+static sds getClientOutputBuffer(client *c) {
+    sds cmd_response = sdsempty();
+    listIter li;
+    listNode *ln;
+    clientReplyBlock *val_block;
+    listRewind(c->reply,&li);
+    
+    serverAssert(c->bufpos == 0);
```

Comment:
I think it might not be clear why `c->bufpos` would be `0` here. We should call out this method should only be used for fake recording client where the statically allocated block isn't used.